### ISSUE TITLE
Navbar Not Keyboard Accessible

### ISSUE THREAD

**Issue by ajdubovoy** [lewagon/rails-templates#151]
In the `_navbar.html.erb` partial we have:

```erb
            <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
```

This means that all the click events are on an image, which does not receive keyboard focus and is not accessible to either keyboard or screen-reader users. Also the image has no `alt` text.

It should most likely be wrapped in a `<button>` element (styled to not change the look of the image, when not focused), and the image should be given `alt` text.

Thanks so much!

**Comment by Eschults** [comment]
Hello @ajdubovoy 👋 

Thank you for the improvement suggestion, this makes sense indeed 👍  
Would you happen to have it implemented w/ a student already and would care to submit a PR on https://github.com/lewagon/awesome-navbars 🙏 ? I'd be happy to review it 👌

**Comment by ajdubovoy** [comment]
Hi @Eschults please excuse my delay! Looks like I don't have push access to that repo. But here's the fix:

navbar_wagon:

```erb
<div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
  <div class="container-fluid">
    <%= link_to "#", class: "navbar-brand" do %>
      <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
    <% end %>

    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav me-auto">
        <% if user_signed_in? %>
          <li class="nav-item active">
            <%= link_to "Home", "#", class: "nav-link" %>
          </li>
          <li class="nav-item">
            <%= link_to "Messages", "#", class: "nav-link" %>
          </li>
          <li class="nav-item dropdown">
          <a href="#" class="avatar dropdown-toggle" id="navbarDropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
            <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar" %>
          </a>
            <div class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdown">
              <%= link_to "Action", "#", class: "dropdown-item" %>
              <%= link_to "Another action", "#", class: "dropdown-item" %>
              <%= link_to "Log out", destroy_user_session_path, data: {turbo_method: :delete}, class: "dropdown-item" %>
            </div>
          </li>
        <% else %>
          <li class="nav-item">
            <%= link_to "Login", new_user_session_path, class: "nav-link" %>
          </li>
        <% end %>
      </ul>
    </div>
  </div>
</div>
```

navbar_wagon_without_login:

```erb
<div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
  <div class="container-fluid">
    <%= link_to "#", class: "navbar-brand" do %>
      <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
    <% end %>

    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav me-auto">
        <li class="nav-item active">
          <%= link_to "Home", "#", class: "nav-link" %>
        </li>
        <li class="nav-item">
          <%= link_to "Messages", "#", class: "nav-link" %>
        </li>
        <li class="nav-item dropdown">
          <a href="#" class="avatar dropdown-toggle" id="navbarDropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
            <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar" %>
          </a>
          <div class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdown">
            <%= link_to "Action", "#", class: "dropdown-item" %>
            <%= link_to "Another action", "#", class: "dropdown-item" %>
          </div>
        </li>
      </ul>
    </div>
  </div>
</div>
```

**Comment by MarcelSF** [comment]
@ajdubovoy In your solution, should I add an `alt` for the images as well?

**Comment by ajdubovoy** [comment]
Ah yes thank you! I wasn't looking at that part haha
